litmus-portal/frontend/src/components/PredifinedWorkflows/data.ts

```diff
@@ -5,10 +5,10 @@ export default [
     urlToIcon:
       'https://hub.litmuschaos.io/api/icon/1.7.0/generic/node-cpu-hog.png',
     chaosWkfCRDLink:
-      'https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/workflows/node-cpu-hog/workflow.yaml',
+      'https://raw.githubusercontent.com/ishangupta-ds/chaos-workflows/master/Argo/argowf-native-node-cpu-hog.yaml',
```

Comment:
No for the time being it will be this, me and @ksatchit are working on raising a PR for the same, will be replacing username with litmuschaos org once the updated workflows are in the org, currently they are in my repo. @chandankumar4

Reply:
okay! So for the time being can you push this changes to other branch in chaos-chart